docs/src/sdp/api.rst

```diff
@@ -65,6 +65,19 @@ SLR83
 ..       Probably need some policy on shat lives in main folder vs configs.
 ..       To control the number of processors we support.
 
+Lhotse processors
+#################
+
+The following processors leverage `lhotse`_, a speech data handling library that contains
```

Comment:
@pzelasko "Lhotse" should be capitalized?

Reply:
Hmmm I can never decide. Technically the name is `lhotse` (and other projects from this family are lowercase too, e.g. `k2`, `icefall`, `sherpa`) but sometimes my finger just sub-conciously hits shift 🙃 Let me change that though so it doesn't strike anybody as odd.